docs/design/downwardapi_resources_limits_requests.md

```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
-<strong>
-The latest release of this document can be found
-[here](http://releases.k8s.io/release-1.3/docs/design/downward_api_resources_limits_requests.md).
-
```

Comment:
Munger script removed it and verify-munge passes as well, not sure what to make of it